crates/weaver_resolver/src/registry.rs

```diff
@@ -380,9 +376,10 @@ mod tests {
             }
 
             let mut attr_catalog = AttributeCatalog::default();
-            let observed_registry = resolve_registry(
-                unresolved_registry_from_specs("https://semconv-registry.com", &sc_specs),
+            let observed_registry = resolve_semconv_registry(
                 &mut attr_catalog,
+                "https://semconv-registry.com",
```

Comment:
Is there an http address or IP address we can use with guaranteed failure that won't ever be domain-squatted here?

Reply:
For this unit test, I will use a local IP address. Thanks